test/e2e/upgrade/alert/alert.go

```diff
@@ -86,7 +86,13 @@ func (t *UpgradeTest) Test(f *framework.Framework, done <-chan struct{}, upgrade
 			},
 		},
 	}
-
+	allowedFiringAlerts := helper.MetricConditions{
+		{
+			// TODO: Add all the control-plane components once all of them have PDBs configured
+			Selector: map[string]string{"alertname": "PodDisruptionBudgetAtLimit", "namespace": "openshift-etcd"},
+			Text:     "Excluded because PodDisruptionBudgetAtLimit is a warning level that is bound to trigger during upgrades as we upgrade one node at a time and desired number of healthy pods match current number of healthy pods as maxUnavailable is set to 1",
```

Comment:
i'm ok with this as a fast fix to get CI running again, but i don't consider this a fix for the fundamental bug(that the alert fires during upgrade) because we are trying to reduce the number of alerts that fire during upgrades because it causes customer pain/confusion.

we need to find a way to avoid this alert entirely during upgrades, not just mask it in our e2e.

Reply:
I think we should revert the alert for now, instead of working around it here.  Alerts shouldn't fire during normal upgrades

/hold

Reply:
Revert in https://github.com/openshift/cluster-kube-controller-manager-operator/pull/533